### PR TITLE
Micah Nichols

### PR DESCRIPTION
**Pull request by thejanjan**

### CODE REVIEW

File: brevets/acp_times.py [hunk 1]
```diff
@@ -4,14 +4,38 @@
...
-#  with these signatures. You must keep
-#  these signatures even if you don't use all the
-#  same arguments.
-#
+def seconds(hour = 0, minute = 0, second = 0):
+    """Returns the number of seconds given other time arguments."""
+    return hour * 3600 + minute * 60 + second
+
+
+# A dictionary mapping the control locations
```
Comment (ElizabethB-st) [inline brevets/acp_times.py:17]: Great! This is very clear the way you have it organized

File: brevets/tests/test_acp_times.py [hunk 2]
```diff
@@ -4,8 +4,72 @@
...
+
+
+def test_closing_c():
+    assert close_time(0,    1000, arrow.get(0)) == arrow.get(seconds(hour=1))
+    assert close_time(400,  1000, arrow.get(0)) == arrow.get(seconds(hour=26, minute=40))
+    assert close_time(600,  1000, arrow.get(0)) == arrow.get(seconds(hour=40, minute=0))
+    assert close_time(890,  1000, arrow.get(0)) == arrow.get(seconds(hour=65, minute=23))
+    assert close_time(1000, 1000, arrow.get(0)) == arrow.get(seconds(hour=75, minute=0))
+
+
```
Comment (ElizabethB-st) [inline brevets/tests/test_acp_times.py:69]: Good job checking for edge cases

**Review by ElizabethB-st — COMMENTED** [review]
Good job Micah!
Your code was very well written and easy to follow. And your comments throughout your code were great, they helped me understand your thought process.
